# drone geo fence for outdoor flight
DroneGeoFence:
  x_min: -50.0
  x_max:  50.0
  y_min: -50.0
  y_max:  50.0
  z_min:  0.0
  z_max:  20.0